src/Handler/Diff.php

```diff
@@ -126,13 +129,18 @@ public function getErrorOutput()
         return $this->errorOutput;
     }
 
-    private function copyToTmpLocation($templatePath)
+    private function createTmpEnv($templatePath, $tmpPath)
     {
-        $tmpPath = manala_get_tmp_dir('diff_');
-
         $this->fs->mkdir($tmpPath);
-        $this->fs->mirror($templatePath, $tmpPath);
-        $this->fs->remove("$tmpPath/manala.yml");
+
+        $env = EnvFactory::createFromMetadata($this->envType, MetadataParser::parse($this->envType, $this->cwd.'/ansible/.manalize.yml'));
+        $dump = Dumper::dump($env, $tmpPath);
+
+        do {
+            $dump->next();
+        } while ($dump->valid());
```

Comment:
```php
for (; $dump->valid(); $dump->next());
```

or: 

```php
foreach ($dump as $̄); // Yes it works
```

is way more fun 😄 

(See https://twitter.com/nicolasgrekas/status/586108028421804032 about the variable name in the foreach :D )